Configuration/Settings.yaml

```
@@ -2,7 +2,7 @@ Neos:
   Flow:
     http:
       middlewares:
-        redirect:
+        graphQlOptions:
```

Good one, what about:
```suggestion
        'Wwwision.GraphQL:Options':
```

IMO it would be good practice to use a FQN for 3rd party middlewares – wdyt?